packages/react/src/components/Dashboard/DashboardGrid.mdx

```diff
@@ -0,0 +1,273 @@
+| layouts.xl[].x  | number          |         | The x position of this card at the xl breakpoint  |
+| layouts.xl[].y  | number          |         | The y position of this card at the xl breakpoint  |
+| layouts.xl[].w  | number          |         | The width of this card at the xl breakpoint       |
+| layouts.xl[].h  | number          |         | The height of this card at the xl breakpoint      |
+| layouts.lg      | arrayOf(object) |         |                                                   |
+| layouts.lg[].i  | string          |         | the id of this element                            |
+| layouts.lg[].x  | number          |         | The x position of this card at the lg breakpoint  |
+| layouts.lg[].y  | number          |         | The y position of this card at the lg breakpoint  |
+| layouts.lg[].w  | number          |         | The width of this card at the lg breakpoint       |
+| layouts.lg[].h  | number          |         | The height of this card at the lg breakpoint      |
+| layouts.md      | arrayOf(object) |         |                                                   |
+| layouts.md[].i  | string          |         | the id of this element                            |
+| layouts.md[].x  | number          |         | The x position of this card at md breakpoint      |
+| layouts.md[].y  | number          |         | The y position of this card at md breakpoint      |
+| layouts.md[].w  | number          |         | The width of this card at md breakpoint           |
+| layouts.md[].h  | number          |         | The height of this card at md breakpoint          |
+| layouts.sm      | arrayOf(object) |         |                                                   |
+| layouts.sm[].i  | string          |         | the id of this element                            |
+| layouts.sm[].x  | number          |         | The x position of this card at the sm breakpoint  |
+| layouts.sm[].y  | number          |         | The y position of this card at the sm breakpoint  |
+| layouts.sm[].w  | number          |         | The width of this card at the sm breakpoint       |
+| layouts.sm[].h  | number          |         | The height of this card at the sm breakpoint      |
+| layouts.xs      | arrayOf(object) |         |                                                   |
+| layouts.xs[].i  | string          |         | the id of this element                            |
+| layouts.xs[].x  | number          |         | The x position of this card at the xs breakpoint  |
+| layouts.xs[].y  | number          |         | The y position of this card at the xs breakpoint  |
+| layouts.xs[].w  | number          |         | The width of this card at the xs breakpoint       |
+| layouts.xs[].h  | number          |         | The height of this card at the xs breakpoint      |
+
+## Source Code
+
+[Source code](https://github.com/carbon-design-system/carbon-addons-iot-react/tree/next/packages/react/src/components/DashboardGrid)
```

Comment:
Should be https://github.com/carbon-design-system/carbon-addons-iot-react/blob/next/packages/react/src/components/Dashboard/DashboardGrid.jsx